server_code/authorisation.py

```diff
@@ -7,9 +7,28 @@
 import functools
 
 import anvil.users
+from anvil.tables import app_tables
 
 __version__ = "2.6.0"
 
+config = {"get_roles_row": lambda user: anvil.users.get_user()["roles"]}
+
+
+def set_config(**kwargs):
+    if "get_roles_row" in kwargs:
+        set_user_roles_getter(kwargs["get_roles_row"])
+
+
+def set_user_roles_getter(option):
```

Comment:
let's make this private since it's not part of the public api

```suggestion
def set_config(**kwargs):
    if "get_roles" in kwargs:
        _set_user_roles_getter(kwargs["get_roles"])


def _set_user_roles_getter(option):
```